programs/github

```diff
@@ -79,6 +79,30 @@ if flags.branch and flags.tag then
 	return printError('--branch and --tag cannot both be specified')
 end
 
+
+local function hasEnoughSpace(repoSize, freeSpace)
+	local sizeError = 0.2
+
+	local function warnAndContinue()
+		print("Repository maybe to large to download, attempt anyway?")
```

Comment:
A better message:

    write("Repository may be too large to download, attempt anyway? [Y/n]: ")